### PR TITLE
Add VERSION file.

### LINKED ISSUE #17: Request: Add VERSION file
It would be nice to have a VERSION file in the release master branch that includes the current releases info.  I was thinking a json file that includes version, date and build number something like:

{
    "decred": {
        "version": "0.1.2",
        "build": "01",
        "date": "2016-05-03"
    },
    "paymetheus": {
        "version": "0.0.8"
    },
    "miners": {
        "version": "0.1.0"
    }
}

This would provide all the info needed to create automated tools to check for and update the different software being released.

### PR DESCRIPTION
**Pull request by jcvernaleo**
This provides a machine readable (json) file which
lists the latest VERSION and release dates for each
group of software in this repository.

Closes #17

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "decred": {
+    "version": "0.1.2",
+    "build": "01",
+    "date": "2016-05-03"
+  },
+  "paymetheus": {
```
Comment (jrick) [inline VERSION]: Paymetheus is typically capitalized.

**Comment by jcvernaleo** [comment]
@jrick capitalization fixed

**Comment by jcvernaleo** [comment]
Added url field on @jrick's suggestion.

**Comment by jrick** [comment]
ok, as long as we remember to keep this up to date.

**Comment by jcvernaleo** [comment]
@jrick The only reason I don't worry too much about keeping it up-to-date is that the releases are already pretty checklist based anyway so adding one more item to the checklist isn't a huge deal.

**Comment by alexlyp** [comment]
OK

**Pull request merged**